### PR TITLE
Implement "EntityEquipment" (held item and armor slots) for MCPE clients

### PR DESCRIPTION
**Pull request by MrPowerGamerBR**

### CODE REVIEW

File: src/protocolsupport/protocol/packet/middleimpl/clientbound/play/v_pe/EntityEquipment.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+				case (5): {
+					dataCache.setHelmet(this.itemstack);
+					break;
+				}
+			}
+			ItemStackSerializer.writeItemStack(serializer, ProtocolVersion.MINECRAFT_PE, this.cache.getLocale(), dataCache.getHelmet(), true); // helmet
+			ItemStackSerializer.writeItemStack(serializer, ProtocolVersion.MINECRAFT_PE, this.cache.getLocale(), dataCache.getChestplate(), true); // chestplate
+			ItemStackSerializer.writeItemStack(serializer, ProtocolVersion.MINECRAFT_PE, this.cache.getLocale(), dataCache.getLeggings(), true); // leggings
+			ItemStackSerializer.writeItemStack(serializer, ProtocolVersion.MINECRAFT_PE, this.cache.getLocale(), dataCache.getBoots(), true); // boots
+
```
Comment (7kasper) [inline src/protocolsupport/protocol/packet/middleimpl/clientbound/play/v_pe/EntityEquipment.java]: No need for comments. It's in the name :)

File: src/protocolsupport/protocol/utils/types/NetworkEntity.java [hunk 2]
```diff
@@ -115,7 +124,29 @@ public void setPeBaseFlag(int bitpos, long value) {
...
+		public void setLeggings(ItemStackWrapper leggings) {
+			this.leggings = leggings;
+		}
+
+		public ItemStackWrapper getLeggings() { return this.leggings; }
+
+		public void setBoots(ItemStackWrapper boots) {
+			this.boots = boots;
+		}
+
```
Comment (7kasper) [inline src/protocolsupport/protocol/utils/types/NetworkEntity.java]: Be consistant. Put all getters and setters on one line or all not..

Comment (MrPowerGamerBR) [inline src/protocolsupport/protocol/utils/types/NetworkEntity.java]: Blame IntelliJ, sometimes it likss to add a new line for no reason at all, I will fix that soon

**Comment by 7kasper** [comment]
From telly at airport :small_airplane: ... Please check the mcpemeta branch :smile:

**Pull request merged**